Description: Drop features and deps only useful when being a rust dependency
 introduced in upstream commit a32845f0215391f118a674868a65d78adbd982d0.
 With the implicit compiler_builtin feature enabled, benches fails to build:
 	undefined reference to `core::cmp::min_by'
 With disable-benches.patch it's actually irrelevant, but let's be safe.
Author: Peter Michael Green <plugwash@debian.org>
Forwarded: not-needed
Last-Update: 2024-09-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,16 +40,2 @@
 
-[dependencies.compiler_builtins]
-version = "0.1"
-optional = true
-
-[dependencies.core]
-version = "1.0"
-optional = true
-package = "rustc-std-workspace-core"
-
-[dependencies.std]
-version = "1.0"
-optional = true
-package = "rustc-std-workspace-std"
-
 [features]
@@ -57,6 +43 @@
 no_std = []
-rustc-dep-of-std = [
-    "std",
-    "core",
-    "compiler_builtins",
-]
